Hymns/Home/HomeView.swift

```diff
@@ -32,13 +32,23 @@ struct HomeView: View {
             } else if viewModel.state == .empty {
                 Text("Did not find any songs matching:\n\"\(viewModel.searchParameter)\".\nPlease try a different request").padding().multilineTextAlignment(.center).maxSize(alignment: .center)
             } else {
-                List(viewModel.songResults) { songResult in
-                    NavigationLink(destination: songResult.destinationView) {
-                        SongResultView(viewModel: songResult)
-                    }.onAppear {
-                        self.viewModel.loadMore(at: songResult)
+                if viewModel.songResults.isEmpty {
+                    Spacer()
+                    HStack {
+                        Spacer()
+                        Image("search illustration")
+                        Spacer()
                     }
-                }.resignKeyboardOnDragGesture()
+                    Spacer()
+                } else {
```

Comment:
I think there are probably better ways to center an image other than using Spacers and HStacks

Reply:
So the reason for this is because the alignment left we have on the parent view keeps overriding this views alignment. I tried two other options of putting the Image alignment to center but it still ends up aligned left. Also, I tried .position(x: UIScreen.main.bounds.width/2) but that was also never in the right position. The one other thing we could do is geometry reader but that is probably going to be uglier than the HStack with spacers... Lmk what you think?

Reply:
What if you just did Image(...).maxSize()?

Reply:
Lol! Nice... I didn't think about that. That works pretty well here but the only problem is when it is in landscape mode. It runs up into the search bar to reach it's max size. In regular orientation this would have worked well though.

Reply:
I'm looking at the way you did it, and it looks like it also pushes up the search bar in landscape. Am I missing something here?
![Simulator Screen Shot - iPhone 8 - 2020-05-22 at 17 15 34](https://user-images.githubusercontent.com/1427524/82717154-e8c81e80-9c4f-11ea-9af9-8942b6c51e65.png)

Reply:
True.... in my new commit, I added the image to be able to resize while maintaining its aspect ratio.